src/main/kotlin/sliv/tool/scene/view/virtualizedfx/VirtualizedFXGridProvider.kt

```diff
@@ -5,26 +5,45 @@ import io.github.palexdev.mfxcore.collections.ObservableGrid
 import io.github.palexdev.virtualizedfx.enums.ScrollPaneEnums
 import io.github.palexdev.virtualizedfx.grid.VirtualGrid
 import io.github.palexdev.virtualizedfx.utils.VSPUtils
-import javafx.scene.Node
+import javafx.beans.property.DoubleProperty
+import javafx.geometry.Orientation
 import sliv.tool.scene.model.VisualizationFrame
 import sliv.tool.scene.view.*
+import tornadofx.onChange
 
 object VirtualizedFXGridProvider : GridProvider {
     override fun createGrid(
         data: List<VisualizationFrame>,
         columnsNumber: Int,
         cellWidth: Double,
         cellHeight: Double,
+        scale: DoubleProperty,
         cellFactory: (VisualizationFrame) -> FrameView
-    ): Node {
+    ): Grid {
         val gridData = ObservableGrid.fromList(data, columnsNumber)
         val grid = VirtualGrid(gridData) { item -> FrameViewAdapter(cellFactory(item)) }
         grid.cellSize = Size(cellWidth, cellHeight)
 
         val vsp = VSPUtils.wrap(grid)
         vsp.layoutMode = ScrollPaneEnums.LayoutMode.COMPACT
         vsp.isAutoHideBars = true
+        vsp.isSmoothScroll = true
+        VSPUtils.setVSpeed(vsp, 100.0, 100.0, 100.0)
```

Comment:
why is this required? maybe some comment?

Reply:
Vertical scrolling is too fast by default. It's needed to be tuned to achieve smooth scrolling